test/browser-main.js

```diff
@@ -0,0 +1,6 @@
+const requireAll = requireContext => requireContext.keys().forEach(requireContext);
+
+if (typeof window !== 'undefined') {
```

Comment:
I'd rather use Mocha's `--exclude` option (https://github.com/mochajs/mocha/pull/3210) once it lands instead of this.

Reply:
Mocha has dropped older node versions so we shouldn’t be upgrading too far on it; this approach is fine.

Reply:
Fair enough 👍

Reply:
that is true; but that is something i'm still working on fixing :-)